model/docs/example_with_datasets.qmd

```diff
@@ -302,15 +302,94 @@ out = hosp_model.plot_posterior(
 )
 ```
 
+We can use [ArviZ](https://www.arviz.org/) to visualize the results. Convert the fitted model to Arviz InferenceData object:
+```{python}
+# | label: convert-inferenceData
+import arviz as az
+idata = az.from_numpyro(hosp_model.mcmc)
+```
+Below we plot $90 \%$ and $50 \%$ highest density interval for latent hospital admissions using [plot_hdi](https://python.arviz.org/en/stable/api/generated/arviz.plot_hdi.html):
+
+```{python}
+# | label: fig-output-admission-distribution
+# | fig-cap: Hospital Admissions posterior distribution
+x_data = idata.posterior['latent_hospital_admissions_dim_0']
+y_data = idata.posterior['latent_hospital_admissions']
+
+fig, axes = plt.subplots(figsize=(6,5))
+az.plot_hdi(
+    x_data,
+    y_data,
+    hdi_prob=0.9,
+    color='C0',
+    smooth=False,
+    fill_kwargs={'alpha':0.3},
+    ax=axes,
+)
+
+az.plot_hdi(
+    x_data,
+    y_data,
+    hdi_prob=0.5,
+    color='C0',
+    smooth=False,
+    fill_kwargs={'alpha':0.6},
+    ax=axes,
+)
+
+#Add mean of the posterior to the figure
+mean_latent_hosp_admission = az.summary(idata.posterior['latent_hospital_admissions'])['mean']
+axes.plot(x_data,mean_latent_hosp_admission, color='C0', label='Mean')
+axes.legend()
+axes.set_xlabel('Time', fontsize=10)
+axes.set_ylabel('Posterior distribution (hospital admissions)',fontsize=10);
+```
```

Comment:
I see  `arviz - WARNING - Shape validation failed: input_shape: (1, 2000), minimum_shape: (chains=2, draws=4)`. Can we omit this (prefer to just silence the warning, but we could also sample multiple chains).

Reply:
I can omit the warning without sampling multiple chains by computing the mean using `np.mean` instead of `az.summary` (guess I was being lazy).